2010-08-12  Jie Zhang  <jie@codesourcery.com>

	Backport from mainline:

	gcc/testsuite/
	2010-08-12  Jie Zhang  <jie@codesourcery.com>
	* gcc.dg/graphite/interchange-9.c (M): Define to be 111.
	(N): Likewise.
	(main): Adjust accordingly.

 2010-08-05  Julian Brown  <julian@codesourcery.com>
 
 	Backport from mainline (candidate patch):

=== modified file 'gcc/testsuite/gcc.dg/graphite/interchange-9.c'
--- old/gcc/testsuite/gcc.dg/graphite/interchange-9.c	2010-02-07 19:49:26 +0000
+++ new/gcc/testsuite/gcc.dg/graphite/interchange-9.c	2010-08-20 16:32:45 +0000
@@ -5,8 +5,8 @@
 #include <stdio.h>
 #endif
 
-#define N 1111
-#define M 1111
+#define N 111
+#define M 111
 
 static int __attribute__((noinline))
 foo (int *x)
@@ -38,7 +38,7 @@
   fprintf (stderr, "res = %d \n", res);
 #endif
 
-  if (res != 2468642)
+  if (res != 24642)
     abort ();
 
   return 0;

